documentation/null-handling/README.md

```diff
@@ -0,0 +1,35 @@
+## Handling of null values
+
+In the code we should annotate all parameters of methods and constructors with the `@Nullable` or `@NonNull`
```

Comment:
In general I'm with you but https://github.com/hashgraph/hedera-services/pull/5483 already changes that by provide our own common Utility for that. So we should wait until that is merged and than update the documentation.
Plus: since we now have 2 repos become 1 we need to discuss how we handle documentation anyway, see my comment [here](https://github.com/hashgraph/hedera-services/pull/5565#issuecomment-1463442397).